generic3g/OutputInfo.F90

```diff
@@ -223,7 +223,7 @@ function make_ungridded_dim(info, n, rc)
       type(UngriddedDim) :: make_ungridded_dim
       integer, intent(in) :: n
       type(ESMF_Info), intent(in) :: info
-      integer, optional, intent(out) :: rc 
+      integer, optional, intent(out) :: rc
       integer :: status
       character(len=:), allocatable :: key
```

Comment:
Looks unused.

Reply:
I didn't make this particular file, from previous PR. But that rc is used, there's lots of `_RC` in the procedure.

Reply:
Used

Reply:
I didn't make this particular file, from previous PR. But that rc is used, there's lots of `_RC` in the procedure.